modules/tournament/src/main/Tournament.scala

```diff
@@ -102,27 +111,135 @@ case class Tournament(
 
   def pairingsClosed = secondsToFinish < pairingsClosedSeconds
 
+  //start at 0 as it's actaully an index for medley variants in front end
   def medleyRound: Option[Int] =
-    if (isStarted) medleyMinutes.map(_ * 60).map((nowSeconds - startsAt.getSeconds).toInt / _)
-    else None
+    if (isStarted) {
+      medleyIntervalSeconds
+        .map(
+          _.map { var s = 0; t => { s += t; s } }
```

Comment:
I dont get how this works. I'm assuming it takes a list of say:
[10,10,10,10,5]
and converts it to:
[10,20,30,40,45]?

Reply:
yes that's correct.